### ISSUE TITLE
Version 4.2.59.0  preview pane can't display xlsx content

### ISSUE THREAD

**Issue by wordpure** [dnGrep/dnGrep#1181]
After updating to 4.2.59.0, the preview pane cannot display the content of xlsx. 

![图片](https://github.com/user-attachments/assets/7f29a8a0-604a-43fd-b98c-357501ac7251)

Preview Pane Error

```log
Error opening the file: Could not find file 'C:\Users\xxx\AppData\Local\Temp\dnGrep-files\dnGREP-OpenXML\test.xlsx_0_[Sheet1]_33BAB6E4F42DC6709D6E6D30536C943646BD596229441C1F75619527E 2F735B6.txt'.
```

<details>
<summary>Grep_Error_Log.xml</summary>

Error opening the file: Could not find file 'C:\Users\xxx\AppData\Local\Temp\dnGrep-files\dnGREP-OpenXML\test.xlsx_0_[Sheet1]_33BAB6E4F42DC6709D6E6D30536C943646BD596229441C1F75619527E2F735B6.txt'.

```xml
<logevent time="2024-08-21 23:00:55.3847" level="Info" logger="dnGREP.WPF.MainViewModel">
  <message>Search completed in 0.318s – 0 matches found in 1 files of 1 searched. Search for: 	0.318s	0	1	1</message>
</logevent>
<logevent time="2024-08-21 23:00:59.5645" level="Info" logger="dnGREP.Common.GrepCore">
  <message>Maximum concurrent tasks used in search: 1</message>
</logevent>
<logevent time="2024-08-21 23:00:59.5645" level="Info" logger="dnGREP.WPF.MainViewModel">
  <message>Search completed in 0.166s – 1 matches found in 1 files of 1 searched. Search for: lorem	0.166s	1	1	1</message>
</logevent>
```

</details>

**Comment by doug24** [comment]
Thanks for the report, yes this is a bug.  

In the file name, the digit between .xlsx and the sheet name should be the sheet number, starting from 1.  But the file name it's trying to open is always zero.

**Comment by doug24** [comment]
Fixed in [4.2.84](https://github.com/dnGrep/dnGrep/releases/tag/v4.2.84.0)

**Comment by wordpure** [comment]
After the update xlsx displays the content properly in the preview pane, thanks for your work🌹
